storage/innobase/srv/srv0srv.cc

```diff
@@ -1479,6 +1485,29 @@ srv_export_innodb_status(void)
 
 	export_vars.innodb_buffer_pool_pages_dirty = flush_list_len;
 
+	export_vars.innodb_buffer_pool_pages_made_young = stat.n_pages_made_young;
+	export_vars.innodb_buffer_pool_pages_made_not_young = stat.n_pages_not_made_young;
+	export_vars.innodb_buffer_pool_pages_old = 0;
+	for (i = 0; i < srv_buf_pool_instances; i++) {
```

Comment:
I prefer minimizing variable scopes. Please use the C++98 style:
```c++
for (ulong i = 0; i < srv_buf_pool_instances; i++) {
```
(Yes, `srv_buf_pool_instances` is declared `ulong`, not `ulint`.)